src/icons/DownArrow.tsx

```diff
@@ -0,0 +1,16 @@
+import { SvgIcon } from '@material-ui/core';
+
+export const DownArrow = (props: any) => (
+  <SvgIcon {...props}>
+    <path
+      d="M1 1L7 7L13 1"
+      viewBox="0 0 W H"
+      fill="none"
+      stroke="#516369"
```

Comment:
`stroke="currentColor"`
that way the text color will automatically set this. Works unless multicolor icon